homeassistant/components/google_cloud/tts.py

```diff
@@ -0,0 +1,212 @@
+"""Support for the Google Cloud TTS service."""
+import logging
+import os
+import asyncio
+import async_timeout
+import voluptuous as vol
+from google.cloud import texttospeech
+from homeassistant.components.tts import CONF_LANG, PLATFORM_SCHEMA, Provider
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_KEY_FILE = 'key_file'
+CONF_GENDER = 'gender'
+CONF_VOICE = 'voice'
+CONF_ENCODING = 'encoding'
+CONF_SPEED = 'speed'
+CONF_PITCH = 'pitch'
+CONF_GAIN = 'gain'
+
+SUPPORTED_LANGUAGES = [
+    'en', 'da', 'nl', 'fr', 'de', 'it', 'ja', 'ko', 'nb',
+    'pl', 'pt', 'ru', 'sk', 'es', 'sv', 'tr', 'uk',
+]
+DEFAULT_LANG = SUPPORTED_LANGUAGES[0]
+
+SUPPORTED_GENDERS = [
+    'Neutral', 'Female', 'Male',
+]
+DEFAULT_GENDER = SUPPORTED_GENDERS[0]
+GENDERS_DICT = {
+    'Neutral': texttospeech.enums.SsmlVoiceGender.NEUTRAL,
```

Comment:
Well, I made it similar to MS TTS component:
https://github.com/home-assistant/home-assistant/blob/8fc30569a9d201c96efc12ed3893d5b1a06a14a0/homeassistant/components/microsoft/tts.py#L30
But yeah, it might be better to leave everything lowercase. What do you think?

Reply:
Now it is not case-sensitive.